src/features/default-plugins.js

```diff
@@ -1,22 +1,7 @@
 import { addPlugin } from './plugins';
 
-if (typeof navigator != 'undefined' && navigator.product == 'ReactNative') {
-  // I'm in react-native
-  // Parse '9' to 9. React native uses numbers and not strings for styles.
-  addPlugin('parseKeyValue', (k, v) => {
-    const intRegex = /^([+-]?[1-9]\d*|0)$/;
-    const floatRegex = /^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$/;
-    if (intRegex.test(v)) {
-      return [k, parseInt(v, 10)];
-    } else if (floatRegex.test(v)) {
-      return [k, parseFloat(v)];
-    }
-  });
-} else {
-  // I'm on the web or node js!
-  // CSS:after content fix. Turns 'value' into ''value'';
-  addPlugin(
-    'parseKeyValue',
-    (k, v) => k === 'content' && v === '' && [k, `'${v}'`]
-  );
-}
+// CSS:after content fix. Turns 'value' into ''value'';
```

Comment:
Do we want this plugin to run on React Native? It is a css specific thing.

Reply:
Looking at https://github.com/vhpoet/react-native-styling-cheat-sheet and my code - not sure that `content` is ever used. And :after is not supported any way. If you think is a performance hit we can put an if there for react native otherwise it shouldn't break anything.

Reply:
I'd like to exclude it on react native, not so much for performance, but not to suddenly get weird bugs in the future.